Formula/apache-pulsar.rb

```diff
@@ -0,0 +1,83 @@
+class ApachePulsar < Formula
+  desc "Cloud-native distributed messaging and streaming platform"
+  homepage "https://pulsar.apache.org/"
+  url "https://www.apache.org/dyn/mirrors/mirrors.cgi?action=download&filename=pulsar/pulsar-2.8.0/apache-pulsar-2.8.0-src.tar.gz"
+  mirror "https://archive.apache.org/dist/pulsar/pulsar-2.8.0/apache-pulsar-2.8.0-src.tar.gz"
+  sha256 "0e161a81c62c7234c1e0c243bb6fe30046ec1cd01472618573ecdc2a73b1163b"
+  license "Apache-2.0"
+  head "https://github.com/apache/pulsar.git", branch: "master"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cppunit" => :build
+  depends_on "libtool" => :build
+  depends_on "maven" => :build
+  depends_on "pkg-config" => :build
+  depends_on "protobuf" => :build
+  depends_on arch: :x86_64
+  depends_on "openjdk@11"
+
+  def install
+    chmod "+x", "src/rename-netty-native-libs.sh"
```

Comment:
This should be fixed upstream.

Reply:
It is executable in master: https://github.com/apache/pulsar/blob/master/src/rename-netty-native-libs.sh

The only place it seems to be not executable is in the tarballs on the apache mirrors.

Reply:
Then upstream needs to check why.

Reply:
I filed [an issue](https://github.com/apache/pulsar/issues/11833) upstream and linked to it in the formula.